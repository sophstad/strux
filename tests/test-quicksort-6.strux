int main() {
    string[5] a = ["kofi", "josh", "millie", "sophie", "edwards"];
    a.quickSort();
    a.show();
    return 0;
}