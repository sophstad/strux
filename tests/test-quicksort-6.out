[edwards, josh, kofi, millie, sophie]
